### ISSUE TITLE
[Submission] 1 params - corbensorenson (LS1Art, art/experimental)

### ISSUE THREAD

**Issue by corbensorenson** [anadim/AdderBoard#42]
### Author

corbensorenson + Codex

### Unique Parameter Count

1

### Accuracy

100.00% on 10,000 random pairs + 10 edge cases (seed=2025)

### Method

Hand-coded weights

### Architecture

1-layer decoder-only, d=1, 1 head, local causal self-attention (window=4)

### Key Tricks

- interleaved stream with explicit state slots (`a_i`, `b_i`, `slot_i`)
- dedicated slot token id (`SLOT_TOKEN`) to avoid token overloading
- local causal attention window over `[prev_state, a_i, b_i, slot_i]`
- state selection via squared-distance logits to centers `{0,1,2,3}`
- single counted payload scalar drives embedding and projection scales
- explicit state-token validation in decode path

### Link to Code

https://gist.github.com/corbensorenson/1c6d0a7d67ad529c1e516c354817ff25

### Verification Output

```shell
Model: Codex LS1Art InterleavedStreamSlots
Author: corbensorenson + Codex
Parameters (unique): 1
Architecture: 1-layer decoder-only, d=1, 1 head, local causal self-attention (window=4)
Tricks: interleaved stream with explicit state slots (a_i, b_i, slot_i), dedicated slot token id (no token overloading with bit-0 token), local causal window over [prev_state, a_i, b_i, slot_i], state logits from squared-distance to centers {0,1,2,3}, single counted payload tensor drives embedding + projection scales, art/experimental intent: strict-count boundary probe with fully disclosed structure

  Progress: 1000/10010 (1000/1000 correct) [5.2s]
  Progress: 2000/10010 (2000/2000 correct) [10.8s]
  Progress: 3000/10010 (3000/3000 correct) [15.6s]
  Progress: 4000/10010 (4000/4000 correct) [20.3s]
  Progress: 5000/10010 (5000/5000 correct) [25.0s]
  Progress: 6000/10010 (6000/6000 correct) [29.7s]
  Progress: 7000/10010 (7000/7000 correct) [34.5s]
  Progress: 8000/10010 (8000/8000 correct) [39.2s]
  Progress: 9000/10010 (9000/9000 correct) [44.0s]
  Progress: 10000/10010 (10000/10000 correct) [48.8s]

Results: 10010/10010 correct (100.00%)
Time: 48.8s (205 additions/sec)
Status: QUALIFIED (threshold: 99%)
```

### Additional Notes

This is our **art/experimental submission**.

Intent:
- boundary-probe and rule-clarification submission with full disclosure, not our low-risk primary competitive line.
- disclose a constructive single-scalar technique in a fully auditable transformer-shaped forward path.

Counting declaration (strict):
- alias tying: 1
- raw: 1
- strict tensor raw: 1

Safety declaration:
- no hidden buffers
- no Python-side arithmetic implementation of addition
- fixed autoregressive decode over model logits only
